certified-connectors/Flotiq/apiDefinition.swagger.json

```diff
@@ -0,0 +1,216 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Flotiq headless CMS",
+    "description": "Flotiq headless content management system",
+    "contact":{
+      "name": "Flotiq support",
+      "url": "https://flotiq.com",
+      "email": "hello@flotiq.com"
+    }
+  },
+  "host": "api.flotiq.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/v1/internal/contenttype/{contentTypeId}": {
+      "get": {
+        "summary": "Get Content Type schema",
+        "operationId": "GetContentTypeSchema",
+        "description": "Get Content Type schema",
+        "parameters": [
+          {
+            "name": "contentTypeId",
+            "in": "path",
+            "type": "string",
+            "description": "Select Content Type Definition id you want fields from",
+            "required": true,
+            "x-ms-summary": "Select Content Type",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CTD"
+            }
+          }
+        }
+      }
+    },
+    "/api/v1/content/{contentTypeId}":{
+      "post": {
+        "summary": "Create Content Object",
+        "description": "Create Content Object",
+        "operationId": "CreateContentObject",
+        "consumes": ["application/json"],
+        "parameters": [
+          {
+            "name": "contentTypeId",
+            "default": "",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "description": "Select the CTD",
+            "x-ms-summary": "Select",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "GetCTD",
+              "value-collection": "data",
+              "value-path": "name",
+              "value-title": "label"
+            }
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Dynamic schema of objects in selected CTD",
+            "schema": {
+              "allOf":[
+              {"type":"object",
+                "properties":{
+                  "id":{"type":"string"}
+                }
+              },
+              {
+              "type":"object",
+
+              "x-ms-dynamic-properties":{
+                "operationId": "GetContentTypeSchema",
+                "parameters":{
+                  "contentTypeId": {
+                    "parameterReference": "contentTypeId"
+                  }
+                },
+                "itemValuePath":"schemaDefinition"
+              }
+            }
+            ]
+            },
```

Comment:
@andrzejwp Please fix indentation issues in swagger.

Reply:
Fixed.